### PR TITLE
updated style for author block

### LINKED ISSUE #17: [Author Page] Style Author Page
The author page has in the first section:
* Author Picture
* Author Description

Side by side.
Check multiple author pages to determine if there is any variation.
Choose either:
* The column blocks with a `Columns (Author)` variant
* Specific CSS for the author page in the author page template

To achieve the same look and feel

Screenshot 2023-10-15 at 15 47 27

Example pages (check more author pages):
https://www.servicenow.com/blogs/author/scott-fuller.html
https://www.servicenow.com/blogs/author/karen-pavlin.html

### PR DESCRIPTION
**Pull request by bala-servicenow**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #17 

Before: https://main--servicenow--hlxsites.hlx.live/blogs/author/mary-hain
After: https://issue17--servicenow--hlxsites.hlx.live/blogs/author/mary-hain

Before: https://main--servicenow--hlxsites.hlx.live/blogs/author/abhijit-mitra
After: https://issue17--servicenow--hlxsites.hlx.live/blogs/author/abhijit-mitra

Updated style for existing P tags without making much of a structural change. Also, maintained the responsiveness with specifying width is %. Let me know if this approach is fine as per the standards.

### CODE REVIEW

**Comment by andreituicu** [comment]
@bala-servicenow the following CSS looks broken:

Screenshot 2023-12-01 at 09 34 43

You see the delimiter line that goes over the text when it should be after the whole content.

File: scripts/scripts.js [hunk 1]
```diff
@@ -428,6 +438,7 @@ export function decorateMain(main) {
   decorateH3(main);
   decorateLinkedPictures(main);
   decorateLinks(main);
+  decorateAuthorBlock(main);
```
Comment (andreituicu) [inline scripts/scripts.js:441]: The template specific decorations should be done in the `buildAutoBlocks` function.

We normally create these kind of decorate functions for generic things that are applicable on multiple pages, potentially the whole website.

Comment (bala-servicenow) [reply scripts/scripts.js:441]: I remember, fixing this delimiter issues. Let me look into the code again.

**Comment by bala-servicenow** [comment]
Before: https://main--servicenow--hlxsites.hlx.live/
After: https://issue17--servicenow--hlxsites.hlx.live/blogs/author/mary-hain
https://issue17--servicenow--hlxsites.hlx.live/blogs/author/abhijit-mitra

**Comment by bala-servicenow** [comment]
@andreituicu  - can you please review these changes?

File: blocks/columns/columns.css [hunk 2]
```diff
@@ -30,4 +30,21 @@
     flex: 1;
     order: unset;
   }
+
+  .blog-authors.columns {
```
Comment (andreituicu) [inline blocks/columns/columns.css:34]: The convention that we follow in CSS when styling blocks is: block name followed by block variant (`.columns.blog-authors` - `.${block name}.${block variant}` , following the order that authors put in the word document `Columns (Blog Authors)`.

This helps when the CSS grows to follow the stylings for individual variants.

File: styles/styles.css [hunk 3]
```diff
@@ -366,6 +366,14 @@ main img {
   padding: 0 1.2rem;
 }
 
+.blog-authors.columns > div > div.columns-img-col {
+  padding-bottom: 20px;
+}
+
+.blog-authors.columns > div {
+  font-size: 1.4rem;
+}
```
Comment (andreituicu) [inline styles/styles.css:375]: Is there any reason for this CSS to be in the global `styles/styles.css`? looks like you are only styling only the columns block, so then you can put it in `blocks/columns/columns.css` with the rest of the CSS from this PR.

The rendering logic will make sure to load the columns CSS only if there is a columns block present on the page, thus increasing the performance. The global `styles/styles.css` will always be loaded.

**Review by andreituicu — COMMENTED** [review]
This ones doesn't look yet good:
https://issue17--servicenow--hlxsites.hlx.live/blogs/author/robert-teed
https://issue17--servicenow--hlxsites.hlx.live/blogs/author/farrell-hough

**Comment by andreituicu** [comment]
@bala-servicenow please keep up to date the description of the PR, including the before link. I did it now for you as an example. That's where the aem sync bot looks to see which items to audit and that's where your colleagues should also look to view the before and after when reviewing your PR.

**Pull request merged**